### DISCUSSION TITLE
DAT-System-Info not starting

### DISCUSSION THREAD

**Discussion by quadeqx** [dat-linux/community#62] [Issues]
I encountered an issue regarding the DAT-System-Info in that when I restart my machine it requires me to set up DAT-System-Info, I select [Y] for yes then sets dconky requests me to restart  DAT-System-Info. When I restart it, it requests me to set it up again and the cycle repeats itself and at the end of the day, DAT-System-Info won't start. How can I go about it? Attached is its screenshot
![screen](https://github.com/dat-linux/community/assets/154374993/a2e15b1e-2867-4ec6-91c6-93ca6f491339)

**Comment by dat-linux** [comment]
Hi please update your system. `sudo apt update && sudo apt upgrade` and advise if that sorts it out.

**Comment by quadeqx** [comment]
My system is updated but still the problem persists

**Reply by dat-linux** [reply]
Please uninstall using the admin uninstaller, and try reinstall.

**Comment by quadeqx** [comment]
I have uninstalled and installed Conky and Conky-all but nothing shows up. On the taskbar, the icon (i) has disappeared and in its place a void. Even when I click DAT-System-Info from the control panel it still wouldn't start. I've looked for the dconky but I haven't found it. If you could provide how to reinstall it through the terminal would be quite helpful. Here is a screenshot
![_](https://github.com/dat-linux/community/assets/154374993/93b66e23-3427-4e2a-ad9b-1beb47d7e227)

**Comment by dat-linux** [comment]
What version of control panel are you showing (Help -> About)? THis is a standalone conky called "dconky" and has no relation to a conky you might install on the system. Uninstall it using the DAT Linux control panel uninstaller, then install it again by clicking on one of the Sys Info icons and you'll be prompted to install it.

**Comment by quadeqx** [comment]
plasmashell 5.24.7
